generator.py

```diff
@@ -1,20 +1,25 @@
-from config import IMG_DIR, IMAGE_SIZE, BATCH_SIZE
-from keras.preprocessing.image import ImageDataGenerator
 import random
 import numpy as np
+# configs
+from config import IMG_DIR, IMAGE_SIZE, BATCH_SIZE
+# keras api
+from keras.preprocessing.image import ImageDataGenerator
 
+# const
```

Comment:
you should explain its use not its type

Reply:
they are constant values so we just comment to understand what it is. their usage will be known where they are used

Reply:
when we use uppercase word and underscore between, it is constant, so you dont have to comment like that